tests/unit/helpers.spec.js

```diff
@@ -6,6 +6,7 @@ const exampleData = {
     projects: {
       CDF: {
         owner: 'ABC',
+        category: 'cat1',
```

Comment:
Forgot this data was in the file directly. Might end up moving this out to its own file.

Great job figuring it out anyway :)